### PR TITLE
API change from Eyescale/Equalizer#532

### PR DESCRIPTION
**Pull request by eile**

### CODE REVIEW

File: livre/eq/Channel.cpp [hunk 1]
```diff
@@ -545,8 +545,7 @@ class Channel
 
         try // blend DB frames in computed order
         {
-            eq::Compositor::assembleFramesSorted( dbFrames, _channel, 0,
-                                                  true /* blendAlpha */ );
+            eq::Compositor::blendFrames( dbFrames, _channel, 0 );
```
Comment (bilgili) [inline livre/eq/Channel.cpp]: alpha is also blended automatically ?

**Comment by bilgili** [comment]
+1 ( when Eq is ready )

**Comment by eile** [comment]
Retest this please.

File: livre/eq/Channel.cpp (hunk 1 above)
Comment (eile) [inline livre/eq/Channel.cpp]: Yes, in fact sorting frames without blending is somewhat pointless.

**Comment by eile** [comment]
Retest this please.

**Comment by bilgili** [comment]
Retest this please.

**Comment by eile** [comment]
Retest this please.

**Pull request merged**